client/mysqldump.c

```diff
@@ -6987,6 +7075,10 @@ int main(int argc, char **argv)
   int exit_code;
   int consistent_binlog_pos= 0;
   int have_mariadb_gtid= 0;
+/*
+  MYSQL_ROW row;
+  MYSQL_RES *column_res;
+*/
```

Comment:
What is this comment for? Was it inadvertently left behind from an earlier version of the code?